Syntaxes/Chapel.tmLanguage

```diff
@@ -80,6 +80,17 @@
 			</dict>
 			<key>name</key>
 			<string>string.quoted.double.chapel</string>
+			<key>patterns</key>
+					<array>
+							<dict>
+								<key>include</key>
+								<string>#string_escaped_char</string>
+							</dict>
+							<dict>
+								<key>include</key>
+								<string>#string_placeholder</string>
+							</dict>
+				 </array>
```

Comment:
The indentation on this is a bit funky for the \<array\> portion.  It looks like maybe there are two tabs, or a tab and a space?

Reply:
@lydia-duncan  Yes , sorry . Do I have to make another commit to remove this funky part of the code